props/saber_BC_buttons.h

```diff
@@ -912,6 +928,18 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       if (fusor.angle1() >  M_PI / 3) {
         SaberBase::DoEffect(EFFECT_POWERSAVE, 0);
         return true;
+      } else if (fusor.angle1() < - M_PI / 4) {
+      // pointing down
+        sequential_quote_ = !sequential_quote_;
+        if (sequential_quote_) {
+          beeper.Beep(0.1, 600);
+          beeper.Beep(0.1, 950);
+          beeper.Beep(0.1, 1300);
+          beeper.Beep(0.1, 600);
+        } else {
+          sound_library_.SayRandom();
```

Comment:
How about "random enabled" / "random disabled" ?

Reply:
Love that, thanks.

Reply:
You can move the SayRandom() to before the if statement since it is the same for both branches.

Reply:
Yes, thank you.